intel_extension_for_transformers/llm/runtime/graph/models/model_utils/model_utils.cpp

```diff
@@ -57,14 +57,14 @@
 
 static bool kv_cache_init(const struct model_hparams& hparams, struct model_kv_cache& cache, const ne_type wtype,
                           const int batch_size, const int beam_size) {
-  const int n_head_kv = std::max(hparams.n_head_kv, 1U);
+  const int n_head_kv = 1U;
```

Comment:
I think we don't need the `n_head_kv` var here otherwise it will cause the wrong mem usage. @DDEle can you please help check this modification？